kolibri/core/assets/src/views/SideNav.vue

```diff
@@ -337,11 +337,17 @@
         //  Window size and app context. Changes may need to be made
         // in parallel in both files for non-breaking updates
         // The expected behavior is:
-        // In an app context, on small and medium screens,
-        // show the app Nav
+        // In an app context, on screens with touch capabilities,
```

Comment:
Note, this code comment is meant to be referencing the AppBar.vue, so we will need to make corresponding changes there: https://github.com/learningequality/kolibri/blob/28bfafab626b698896c61420b4b839fa3eb9b139/kolibri/core/assets/src/views/AppBar.vue#L103

Reply:
Oops 😬

Reply:
Would the changes entail removing all instance of breakpoint usage in the app bar or just in places where a similar check is performed(ie. `isAppContext && !isWindowLarge`)? @marcellamaki ?

Reply:
It should just be for the same condition, `isAppContext && !isWindowLarge`. Because really what I should be checking there is just what you've added - is it a touch screen? But I didn't. The rest of the breakpoints in the app bar should not need any changing, at least none that occurs to me right now

Reply:
Maybe https://github.com/learningequality/kolibri/blob/28bfafab626b698896c61420b4b839fa3eb9b139/kolibri/core/assets/src/views/AppBar.vue#L39 ?

Reply:
I think this can actually stay as it is, but I would have to check it locally to confirm

Reply:
Thanks, I have updated the pr with the changes. I also moved the `isTouchDevice` to the sessions getters.